C/quicksort.c

```diff
@@ -0,0 +1,52 @@
+#include <stdio.h>
+
+void swap(int* array, int i, int j) {
```

Comment:
Here it's better passing to arguments the elements instead of array and indices.
Like `void swap(int*a, int*b)`